compat/cpu.h

```diff
@@ -36,23 +36,27 @@ inline int sched_getaffinity(int, size_t, cpu_set_t *) { return ENOTSUP; }
 #endif
 
 #if !defined(CPU_SET)
-#define CPU_SET(cpu, set) (void)0
+inline void CPU_SET(int, cpu_set_t *) {}
```

Comment:
I would totally agree replacing a macro with a macro would 100% make sense if this was a library header that someone else needs to reuse, so that they can #undef them. In practice in our use case we don't really need to, but sure, it's a nit so I'll change that to use trampoline macros